app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -63,6 +63,8 @@ object TelemetryWrapper {
     private const val SEARCHCLEAR_TELEMETRY_VERSION = "2"
     private const val SEARCHDISMISS_TELEMETRY_VERSION = "2"
 
+    private val AFFILIATE_SITES = listOf("Bukalapak", "Tokopedia")
```

Comment:
I think "Bukalapak", "Tokopedia" should not be hard-coded.

Reply:
Fixed